apps/mobile/mobile-app/components/project-details/tab-content/team.tsx

```
@@ -0,0 +1,41 @@
+			name: 'Maria Garcia',
+			role: 'Head of Operations',
+			bio: 'Former school principal with extensive community outreach experience.',
+		},
+	]
+
+	return (
+		<View className="mt-4">
+			<Text className="text-lg font-bold mb-6">Our Team</Text>
+
+			{teamMembers.map((member, index) => (
+				<View key={index} className="flex-row mb-6">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)